File: Converter10_CS/Model/OwnerModel.cs

namespace Converter10.Njc.Model
{

    #region 家主情報モデル

    public class Owdata_Model
    {

        public string Vari_Ow_no { get; set; }
        public string Vari_Kojinhojin_flg { get; set; }
        public string Vari_Ow_name { get; set; }
        public string Vari_Ow_namesjis { get; set; }
        public string Vari_Ow_kana { get; set; }
        public string Vari_Keisyo { get; set; }
        public string Vari_Post_code { get; set; }
        public string Vari_Addr_kenno { get; set; }
        public string Vari_Addr_sino { get; set; }
        public string Vari_Addr_cyo { get; set; }
        public string Vari_Addr_cyome { get; set; }
        public string Vari_Addr_cyomeptn { get; set; }
        public string Vari_Addr_banti { get; set; }
        public string Vari_Addr_etc { get; set; }
        public string Vari_Toukiaddr1 { get; set; }
        public string Vari_Toukiaddr2 { get; set; }
        public string Vari_Tel1 { get; set; }
        public string Vari_Tel2 { get; set; }
        public string Vari_Fax { get; set; }
        public string Vari_Mobiletel1 { get; set; }
        public string Vari_Mobiletel2 { get; set; }
        public string Vari_Mail { get; set; }
        public string Vari_Mobilemail { get; set; }
        public string Vari_Yusentel_kbn { get; set; }
        public string Vari_Yusenmail_kbn { get; set; }
        public string Vari_Birthday { get; set; }
        public string Vari_Nensyu { get; set; }
        public string Vari_Biko_kihon { get; set; }
        public string Vari_Gender { get; set; }
        public string Vari_Honseki { get; set; }
        public string Vari_Kinmu_name { get; set; }
        public string Vari_Kinmu_namesjis { get; set; }
        public string Vari_Kinmu_kana { get; set; }
        public string Vari_Kinmu_postcode { get; set; }
        public string Vari_Kinmu_addr1 { get; set; }
        public string Vari_Kinmu_addr2 { get; set; }
        public string Vari_Kinmu_tel1 { get; set; }
        public string Vari_Kinmu_tel2 { get; set; }
        public string Vari_Kinmu_fax { get; set; }
        public string Vari_Kinmu_gyosyu { get; set; }
        public string Vari_Kinmu_busyo { get; set; }
        public string Vari_Kinmu_nyuryokuym { get; set; }
        public string Vari_Kinmu_nyusyaym { get; set; }
        public string Vari_Url { get; set; }
        public string Vari_Gyosyu { get; set; }
        public string Vari_Daihyo_name { get; set; }
        public string Vari_Daihyo_namesjis { get; set; }
        public string Vari_Daihyo_kana { get; set; }
        public string Vari_Daihyo_yakusyoku { get; set; }
        public string Vari_Daihyo_atenaflg { get; set; }
        public string Vari_Tanto_name { get; set; }
        public string Vari_Tanto_namesjis { get; set; }
        public string Vari_Tanto_kana { get; set; }
        public string Vari_Tanto_busyo { get; set; }
        public string Vari_Tanto_atenaflg { get; set; }
        public string Vari_Sihonkin { get; set; }
        public string Vari_Jugyosu { get; set; }
        public string Vari_Nyuryoku_ym { get; set; }
        public string Vari_Torihikisaki { get; set; }
        public string Vari_Renraku_name { get; set; }
        public string Vari_Renraku_namesjis { get; set; }
        public string Vari_Renraku_kana { get; set; }
        public string Vari_Renraku_keisyo { get; set; }
        public string Vari_Renraku_postcode { get; set; }
        public string Vari_Renraku_addr1 { get; set; }
        public string Vari_Renraku_addr2 { get; set; }
        public string Vari_Renraku_tel1 { get; set; }
        public string Vari_Renraku_tel2 { get; set; }
        public string Vari_Renraku_fax { get; set; }
        public string Vari_Renraku_mobiletel1 { get; set; }
        public string Vari_Renraku_mobiletel2 { get; set; }
        public string Vari_Renraku_yusentelkbn { get; set; }
        public string Vari_Renraku_aidagara { get; set; }
        public string Vari_Biko_renraku { get; set; }
        public string Vari_Sofu_kbn { get; set; }
        public string Vari_Sofu_name { get; set; }
        public string Vari_Sofu_namesjis { get; set; }
        public string Vari_Sofu_kana { get; set; }
        public string Vari_Sofu_keisyo { get; set; }
        public string Vari_Sofu_postcode { get; set; }
        public string Vari_Sofu_addr1 { get; set; }
        public string Vari_Sofu_addr2 { get; set; }
        public string Vari_Sofu_tel1 { get; set; }
        public string Vari_Sofu_tel2 { get; set; }
        public string Vari_Sofu_fax { get; set; }
        public string Vari_Biko_sofu { get; set; }
        public string Vari_Event_tantono { get; set; }
        public string Vari_Useflg { get; set; }
        public string Vari_Findkeyword { get; set; }
        public string Vari_History { get; set; }
        public string Vari_Rowid { get; set; }
        public string Vari_Kaikei_jouhou { get; set; }
        public string Vari_Addr1 { get; set; }            // 20160622 EXEUpdateに伴う修正 (家主住所移行仕様変更) -add
        public string Vari_Addr2 { get; set; }            // 20160622 EXEUpdateに伴う修正 (家主住所移行仕様変更) -add

    }

    #endregion

    #region 家主口座情報モデル

    public class Owdata_koza_Model
    {

        public string Vari_Ow_no { get; set; }
        public string Vari_Ow_kozano { get; set; }
        public string Vari_Kinyu_no { get; set; }
        public string Vari_Kinyu_tenno { get; set; }
        public string Vari_Koza_syubetu { get; set; }
        public string Vari_Koza_bango { get; set; }
        public string Vari_Koza_meigi { get; set; }
        public string Vari_Koza_meigikana { get; set; }
        public string Vari_Yucyokoza_kigo1 { get; set; }
        public string Vari_Yucyokoza_kigo2 { get; set; }
        public string Vari_Yucyokoza_bango { get; set; }
        public string Vari_Biko_koza { get; set; }
        public string Vari_Biko_furikomi { get; set; }
        public string Vari_Sgfirai_kbn { get; set; }
        public string Vari_Sgfirai_no { get; set; }
        public string Vari_Sgfirai_tesufutankbn { get; set; }
        public string Vari_Sgfirai_tesukeisankbn { get; set; }
        public string Vari_Sgfirai_tesukotei1gak { get; set; }
        public string Vari_Sgfirai_tesukotei2gak { get; set; }
        public string Vari_Biko_sgfirai { get; set; }
        public string Vari_History { get; set; }
        public string Vari_Rowid { get; set; }
        public string Vari_Koza_printkbn { get; set; }

    }

    #endregion

    #region 家主イベント情報モデル

    public class Owdata_event_Model
    {

        public string Vari_Ow_no { get; set; }
        public string Vari_Event_kbn { get; set; }
        public string Vari_Event_cnt { get; set; }
        public string Vari_Event_ymd { get; set; }
        public string Vari_Event_data { get; set; }

    }

    #endregion

    #region 家主メモ情報モデル

    public class Owdata_memo_Model
    {

        public string Vari_Ow_no { get; set; }
        public string Vari_Memo_no { get; set; }
        public string Vari_Memo { get; set; }
        public string Vari_History { get; set; }

    }

    #endregion

}

File: Converter10_VB/Model/OwnerModel.vb
Namespace Njc.Model

#Region "家主情報モデル"

    Public Class Owdata_Model

        Public Property Vari_Ow_no As String
        Public Property Vari_Kojinhojin_flg As String
        Public Property Vari_Ow_name As String
        Public Property Vari_Ow_namesjis As String
        Public Property Vari_Ow_kana As String
        Public Property Vari_Keisyo As String
        Public Property Vari_Post_code As String
        Public Property Vari_Addr_kenno As String
        Public Property Vari_Addr_sino As String
        Public Property Vari_Addr_cyo As String
        Public Property Vari_Addr_cyome As String
        Public Property Vari_Addr_cyomeptn As String
        Public Property Vari_Addr_banti As String
        Public Property Vari_Addr_etc As String
        Public Property Vari_Toukiaddr1 As String
        Public Property Vari_Toukiaddr2 As String
        Public Property Vari_Tel1 As String
        Public Property Vari_Tel2 As String
        Public Property Vari_Fax As String
        Public Property Vari_Mobiletel1 As String
        Public Property Vari_Mobiletel2 As String
        Public Property Vari_Mail As String
        Public Property Vari_Mobilemail As String
        Public Property Vari_Yusentel_kbn As String
        Public Property Vari_Yusenmail_kbn As String
        Public Property Vari_Birthday As String
        Public Property Vari_Nensyu As String
        Public Property Vari_Biko_kihon As String
        Public Property Vari_Gender As String
        Public Property Vari_Honseki As String
        Public Property Vari_Kinmu_name As String
        Public Property Vari_Kinmu_namesjis As String
        Public Property Vari_Kinmu_kana As String
        Public Property Vari_Kinmu_postcode As String
        Public Property Vari_Kinmu_addr1 As String
        Public Property Vari_Kinmu_addr2 As String
        Public Property Vari_Kinmu_tel1 As String
        Public Property Vari_Kinmu_tel2 As String
        Public Property Vari_Kinmu_fax As String
        Public Property Vari_Kinmu_gyosyu As String
        Public Property Vari_Kinmu_busyo As String
        Public Property Vari_Kinmu_nyuryokuym As String
        Public Property Vari_Kinmu_nyusyaym As String
        Public Property Vari_Url As String
        Public Property Vari_Gyosyu As String
        Public Property Vari_Daihyo_name As String
        Public Property Vari_Daihyo_namesjis As String
        Public Property Vari_Daihyo_kana As String
        Public Property Vari_Daihyo_yakusyoku As String
        Public Property Vari_Daihyo_atenaflg As String
        Public Property Vari_Tanto_name As String
        Public Property Vari_Tanto_namesjis As String
        Public Property Vari_Tanto_kana As String
        Public Property Vari_Tanto_busyo As String
        Public Property Vari_Tanto_atenaflg As String
        Public Property Vari_Sihonkin As String
        Public Property Vari_Jugyosu As String
        Public Property Vari_Nyuryoku_ym As String
        Public Property Vari_Torihikisaki As String
        Public Property Vari_Renraku_name As String
        Public Property Vari_Renraku_namesjis As String
        Public Property Vari_Renraku_kana As String
        Public Property Vari_Renraku_keisyo As String
        Public Property Vari_Renraku_postcode As String
        Public Property Vari_Renraku_addr1 As String
        Public Property Vari_Renraku_addr2 As String
        Public Property Vari_Renraku_tel1 As String
        Public Property Vari_Renraku_tel2 As String
        Public Property Vari_Renraku_fax As String
        Public Property Vari_Renraku_mobiletel1 As String
        Public Property Vari_Renraku_mobiletel2 As String
        Public Property Vari_Renraku_yusentelkbn As String
        Public Property Vari_Renraku_aidagara As String
        Public Property Vari_Biko_renraku As String
        Public Property Vari_Sofu_kbn As String
        Public Property Vari_Sofu_name As String
        Public Property Vari_Sofu_namesjis As String
        Public Property Vari_Sofu_kana As String
        Public Property Vari_Sofu_keisyo As String
        Public Property Vari_Sofu_postcode As String
        Public Property Vari_Sofu_addr1 As String
        Public Property Vari_Sofu_addr2 As String
        Public Property Vari_Sofu_tel1 As String
        Public Property Vari_Sofu_tel2 As String
        Public Property Vari_Sofu_fax As String
        Public Property Vari_Biko_sofu As String
        Public Property Vari_Event_tantono As String
        Public Property Vari_Useflg As String
        Public Property Vari_Findkeyword As String
        Public Property Vari_History As String
        Public Property Vari_Rowid As String
        Public Property Vari_Kaikei_jouhou As String
        Public Property Vari_Addr1 As String            '20160622 EXEUpdateに伴う修正 (家主住所移行仕様変更) -add
        Public Property Vari_Addr2 As String            '20160622 EXEUpdateに伴う修正 (家主住所移行仕様変更) -add

    End Class

#End Region

#Region "家主口座情報モデル"

    Public Class Owdata_koza_Model

        Public Property Vari_Ow_no As String
        Public Property Vari_Ow_kozano As String
        Public Property Vari_Kinyu_no As String
        Public Property Vari_Kinyu_tenno As String
        Public Property Vari_Koza_syubetu As String
        Public Property Vari_Koza_bango As String
        Public Property Vari_Koza_meigi As String
        Public Property Vari_Koza_meigikana As String
        Public Property Vari_Yucyokoza_kigo1 As String
        Public Property Vari_Yucyokoza_kigo2 As String
        Public Property Vari_Yucyokoza_bango As String
        Public Property Vari_Biko_koza As String
        Public Property Vari_Biko_furikomi As String
        Public Property Vari_Sgfirai_kbn As String
        Public Property Vari_Sgfirai_no As String
        Public Property Vari_Sgfirai_tesufutankbn As String
        Public Property Vari_Sgfirai_tesukeisankbn As String
        Public Property Vari_Sgfirai_tesukotei1gak As String
        Public Property Vari_Sgfirai_tesukotei2gak As String
        Public Property Vari_Biko_sgfirai As String
        Public Property Vari_History As String
        Public Property Vari_Rowid As String
        Public Property Vari_Koza_printkbn As String

    End Class

#End Region

#Region "家主イベント情報モデル"

    Public Class Owdata_event_Model

        Public Property Vari_Ow_no As String
        Public Property Vari_Event_kbn As String
        Public Property Vari_Event_cnt As String
        Public Property Vari_Event_ymd As String
        Public Property Vari_Event_data As String

    End Class

#End Region

#Region "家主メモ情報モデル"

    Public Class Owdata_memo_Model

        Public Property Vari_Ow_no As String
        Public Property Vari_Memo_no As String
        Public Property Vari_Memo As String
        Public Property Vari_History As String

    End Class

#End Region

End Namespace
